src/hooks/useGoogleAnalytics.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+import ReactGA from "react-ga";
+import { EnvHelper } from "../helpers/Environment";
+
+const GA_API_KEY = EnvHelper.getGaKey();
+
+const useGoogleAnalytics = () => {
+  const location = useLocation();
+
+  useEffect(() => {
+    if (GA_API_KEY && GA_API_KEY.length > 1) {
+      ReactGA.initialize(GA_API_KEY);
+      ReactGA.set({ anonymizeIp: true });
+      ReactGA.pageview(location.pathname + location.search);
```

Comment:
ser, also you need `location.hash` to your `pageView` param. This is because we are using IPFS with `/#` in our pathnames.

See screenshot to demonstrate:
![image](https://user-images.githubusercontent.com/80423742/142447408-a2212d86-2bea-4271-b8e8-77eeb4dc065a.png)

Reply:
![image](https://user-images.githubusercontent.com/80423742/142447936-9ade8303-e419-44a3-af86-a5839a58076b.png)